src/dev/src/app/datagrid/expandable-rows/expandable-rows.html

```diff
@@ -91,8 +91,22 @@ <h2>Expandable rows</h2>
             </ng-template>
         </clr-dg-row-detail>
     </clr-dg-row>
-
-
     <clr-dg-footer>{{users.length}} users</clr-dg-footer>
 </clr-datagrid>
 
+<h3>Mixed expandable and non expandable rows</h3>
+
+<clr-datagrid>
+  <clr-dg-column>Name</clr-dg-column>
+  <clr-dg-column>Pokemon</clr-dg-column>
+
+  <clr-dg-row *ngFor="let user of users.slice(0, 4);let i = index">
+    <clr-dg-cell>{{ user.name }}</clr-dg-cell>
+    <clr-dg-cell>{{ user.pokemon.name }}</clr-dg-cell>
+    <clr-dg-row-detail clrIfExpanded *ngIf="i % 2">
```

Comment:
Shouldn't it be `*clrIfExpanded`? If so, can't also do `*ngIf`.

Reply:
It actually is visually okay, at least for the demo purpose. It only does not remove the nodes from the DOM. But it's misleading and you are right - it would better be changed. 
Done! Thanks